.circleci/config.yml

```diff
@@ -25,6 +25,9 @@ jobs:
             - packages/patternfly-4/react-charts/node_modules
             - packages/patternfly-4/react-styles/node_modules
             - packages/patternfly-3/patternfly-react/node_modules
+            - packages/patternfly-3/patternfly-react-wooden-tree/node_modules
```

Comment:
Should we add react-topology while you are in there?

Reply:
No reason not to -- added.